tests/System/SystemTestCase.php

```diff
@@ -49,6 +49,7 @@ public function phpbench($command, $workingDir = '.')
     {
         chdir($this->getWorkingDir($workingDir));
         $bin = __DIR__ . '/../../bin/phpbench --verbose';
+        $bin = str_replace('/', DIRECTORY_SEPARATOR, $bin);
```

Comment:
Hmm, I thought that `/` worked in Windows these days, there are many instances of this. Which version of PHP are you using?